sdk/boms/spring-cloud-azure-dependencies/pom.xml

```diff
@@ -51,15 +51,10 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-sdk-bom</artifactId>
-        <version>1.2.3</version>   <!-- NOTE: This should be updated manually. -->
+        <version>1.2.6</version>   <!-- NOTE: This should be updated manually. -->
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-spring-data-cosmos</artifactId>
-        <version>3.27.0</version> <!-- {x-version-update;com.azure:azure-spring-data-cosmos;dependency} -->
-      </dependency>
```

Comment:
Why delete this?

Reply:
nice catch, i shouldn't have deleted this.